stories/examples/iris-dataset.stories.tsx

```diff
@@ -80,6 +81,7 @@ export function PCA() {
               xAxis="x"
               yAxis="y"
               key={klass}
+              id={klass}
```

Comment:
can we find a better name than "id" ? For me, "id" implies that it cannot be the same for different elements